### ISSUE TITLE
Ideal mode and realistic mode

### ISSUE THREAD

**Issue by Ashwin-droid** [Proto-App/Proto-Android#23]
Components are way too ideal, inductor lacks esr when it is made of magnet wire wrapped around a core, current source has no upper limit for voltage, wire lacks (esr, esc, esl) (which should be user defined and offering a calculator for calculating resistivity won't hurt (with a given gauge rating and material))
Ok this is for digonostic purposes but for proof of concept it is usually not needed to go to such detail so on a flick of a switch in settings to switch to ideal mode will be appreciated.

**Comment by Proto-App** [comment]
Yes, components are ideal. Probably we will add more and more options in components (for capacitor it will be Esr, Esl, Rl etc.)
